src/Compilers/Test/Utilities/CSharp/SyntaxTreeExtensions.cs

```diff
@@ -59,52 +61,35 @@ public static SyntaxTree WithRemoveFirst(this SyntaxTree syntaxTree, string oldT
             return WithReplaceFirst(syntaxTree, oldText, string.Empty);
         }
 
+#if DEBUG
```

Comment:
I kinda think we should keep this in release. It would be useful when trying to debug dumps or in-production code where we don't know the shape of the tree.

Reply:
That's reasonable. Moved the methods into the product code so that it's there for troubleshooting more scenarios.